### DISCUSSION TITLE
Can i use hostinger domain on vercel?

### DISCUSSION THREAD

**Discussion by UnbalancedSkunk** [vercel/community#7250] [Help]
### Summary

Can i use hostinger domain on vercel?

### Steps to Reproduce

no steps

**Comment by amyegan** [comment]
Vercel deployments should work with any custom that has the appropriate DNS records. If you're having trouble with your domain, I recommend checking DNS propagation using a tool like [www.whatsmydns.net](https://www.whatsmydns.net/) or [`dig`](https://en.wikipedia.org/wiki/Dig_(command)). We have some additional domain troubleshooting tips here: https://vercel.com/docs/projects/domains/troubleshooting

If the global DNS records do not match what you have set, then you should [reach out to Hostinger](https://www.hostinger.com/contacts) for help understanding why that is happening. They'll have more information about your domain and Hostinger account.

[a comment by an automated account is not shown]
